src/app/atoms/Button.tsx

```diff
@@ -12,6 +12,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     const { trackEvent } = useAnalytics();
 
     const handleClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+      // TODO: Not send event, if `props.disabled === true`
```

Comment:
onclick event dont fire if button is disabled.. dont understand what is this comment about